deps/crates/crates.gyp

```diff
@@ -49,6 +50,18 @@
           '<(cargo_vendor_dir)/temporal_capi/bindings/cpp',
         ],
       },
+      'conditions': [
+        ['OS=="win"', {
+          'direct_dependent_settings': {
```

Comment:
Well, on a second thought, I think this should be better moved to target `node_crates`. Additionally, with `link_settings`, the `direct_dependent_settings` should be redundant.